pnpm-lock.yaml

```diff
@@ -1107,7 +1107,7 @@ importers:
         version: 3.1.1
       vitest:
         specifier: ^1.2.0
-        version: 1.2.0(@types/node@18.19.3)(lightningcss@1.22.1)
+        version: 1.2.1(@types/node@18.19.3)
```

Comment:
Can you remove the changes to this lock file entirely (probably by copying over the latest lock file from main)?